patches/chrome-browser-ui-webui-browser_command-browser_command_handler.cc.patch

```diff
@@ -0,0 +1,12 @@
+diff --git a/chrome/browser/ui/webui/browser_command/browser_command_handler.cc b/chrome/browser/ui/webui/browser_command/browser_command_handler.cc
+index 2c6fab86614fa3cfce24ef67ac6f0b096c5078e2..bed70179015c68d5f6e9545651cac04c0a7e9a0d 100644
+--- a/chrome/browser/ui/webui/browser_command/browser_command_handler.cc
++++ b/chrome/browser/ui/webui/browser_command/browser_command_handler.cc
+@@ -121,6 +121,7 @@ void BrowserCommandHandler::CanExecuteCommand(
+     case Command::kOpenSafetyCheckFromWhatsNew:
+       can_execute = true;
+       break;
++    default: break;
```

Comment:
why do we need this? Shouldn't we only be calling the superclass if we don't handle it in our subclass?

Reply:
That switch statement doesn't have a `default` clause, so when we extend the enum with our own `Command` values, it results in a compile-time error here.

Reply:
oic, because there is an existing mojo enum we are extending

Reply:
but this patch does not follow standard patching conventions. It should be `BRAVE_CAN_EXECUTE_COMMAND` and this should also include `NOTREACHED_NORETURN`